README.md

```diff
@@ -1,69 +1,293 @@
 ![Plugin Icon](assets/images/brand/github-social-preview.png)
 
 [![Android version and API level](https://img.shields.io/badge/Android-API%20Level%2033-darkgreen.svg)](https://developer.android.com)
-[![Godot version](https://img.shields.io/badge/Godot%20Engine-3.5.2-blue.svg)](https://github.com/godotengine/godot/)
+[![Godot version](https://img.shields.io/badge/Godot%20Engine-3.5.3-blue.svg)](https://github.com/godotengine/godot/)
 [![Google Play Game Services version](https://img.shields.io/badge/Play%20Games%20Services%20v2-17.0.0-green.svg)](https://developers.google.com/games/services/android/quickstart)
 
-> ⚠️ A new version of this plugin for Godot 4.2 can be found [here](https://github.com/Iakobs/godot-play-game-services)!
+> ⚠️ A **Godot 4.x** version of this plugin can be found [here](https://github.com/Iakobs/godot-play-game-services)!
```

Comment:
It is actually a Godot 4.2+ version. I haven't tried with versions between 4 and 4.2, but with 4.2 Godot introduced a change in how android plugins work, that's why I think (I didn't check) the other plugin won't work in versions lower to 4.2

Reply:
Ok!